src/connectors/geo-search/connectGeoSearch.js

```diff
@@ -199,7 +202,7 @@ const connectGeoSearch = (renderFn, unmountFn) => {
       const { state, helper, instantSearchInstance } = initArgs;
       const isFirstRendering = true;
 
-      widgetState.internalToggleRefineOnMapMove = createInternalToggleRefinementonMapMove(
+      widgetState.internalToggleRefineOnMapMove = createInternalToggleFacetRefinementOnMapMove(
```

Comment:
To revert (see https://github.com/algolia/instantsearch.js/pull/3207#discussion_r226257220).